### PR TITLE
[CDE-837] - FilterComponent - html injection. Caja Sanitizer is added.

### PR DESCRIPTION
**Pull request by YuryBY**
@pentaho-nbaker, @pamval  could you please take a look? Thanks.

### CODE REVIEW

File: core-js/src/main/javascript/cdf/components/filter/views/Abstract.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ define([
   '../models/SelectionTree',
   '../../../lib/jquery',
   './scrollbar/ScrollBarFactory',
-  '../HtmlUtils'
-], function (_, Mustache, BaseView, Logger, SelectionTree, $, ScrollBarFactory, HtmlUtils) {
+  'sanitizer'
+], function (_, Mustache, BaseView, Logger, SelectionTree, $, ScrollBarFactory, Sanitezer) {
```
Comment (afrjorge) [inline core-js/src/main/javascript/cdf/components/filter/views/Abstract.js]: heads up for the typo in `Sanitezer`

**Comment by pamval** [comment]
@YuryBY have those new libs been through the new component process?

@afrjorge @jvelasques please review

**Comment by YuryBY** [comment]
@pamval, PR is updated. 
Corresponding COMP case: http://jira.pentaho.com/browse/COMP-3375

**Comment by pamval** [comment]
@afrjorge @jvelasques please review

File: core-js/src/main/javascript/cdf-core-lib-require-js-cfg.js [hunk 2]
```diff
@@ -80,6 +80,20 @@
   //modernizr 2.8.3
   requirePaths['cdf/lib/modernizr'] = prefix + '/modernizr/modernizr-2.8.3';
 
+  requirejs.config({
```
Comment (jvelasques) [inline core-js/src/main/javascript/cdf-core-lib-require-js-cfg.js]: why not follow the same pattern **requirePaths['cdf/lib/libName']** ?

**Comment by pamval** [comment]
@YuryBY devci and cerberus started failing on the js test suite when this went in. 
Are you able to run the javascript test suite locally?

**Pull request merged**